app.js

```diff
@@ -3,7 +3,8 @@ const app = express()
 const {
     getTopics,
     getEndpoints,
-    getArticleByID
+    getArticleByID,
+    getArticles
 } = require('./controllers/nc_news_controllers.js')
 
 app.use(express.json())
```

Comment:
We're not posting things yet, this shouldn't be here. Stick to code that is required for that ticket.